list_of_x: [2.0, 2.0, 0.0, 0.0]
list_of_y: [0.0, 2.0, 2.0, 0.0]
